moveit_ros/planning/robot_model_loader/src/robot_model_loader.cpp

```diff
@@ -131,55 +131,69 @@ void RobotModelLoader::configure(const Options& opt)
         std::string param_name;
         try
         {
-          param_name = prefix + "max_position";
-          double max_position;
-          if (node_->get_parameter(param_name, max_position))
+          param_name = prefix + "has_position_limits";
```

Comment:
If prefix ends with "." than this should work.

Reply:
It ends with a "." see `        std::string prefix =
            rdf_loader_->getRobotDescription() + "_planning.joint_limits." + joint_limit[joint_id].joint_name + ".";`